emigration.lua

```diff
@@ -1,22 +1,29 @@
 --Allow stressed dwarves to emigrate from the fortress
 -- For 34.11 by IndigoFenix; update and cleanup by PeridexisErrant
 -- old version:  http://dffd.bay12games.com/file.php?id=8404
---@module = true
---@enable = true
+--[====[
+emigration
+==========
+Allows dwarves to emigrate from the fortress when stressed,
+in proportion to how badly stressed they are and adjusted
+for who they would have to leave with - a dwarven merchant
+being more attractive than leaving alone (or with an elf).
+The check is made monthly.
 
-local json = require('json')
-local persist = require('persist-table')
+A happy dwarf (ie with negative stress) will never emigrate.
 
-local GLOBAL_KEY = 'emigration' -- used for state change hooks and persistence
+Usage::
 
-enabled = enabled or false
+    emigration enable|disable
+]====]
 
-function isEnabled()
-    return enabled
-end
+enabled = enabled or false
 
-local function persist_state()
-    persist.GlobalTable[GLOBAL_KEY] = json.encode({enabled=enabled})
+local args = {...}
+if args[1] == "enable" then
+    enabled = true
+elseif args[1] == "disable" then
+    enabled = false
 end
```

Comment:
this part looks like it was accidentally copied from an earlier version. Could you fix the merge?

Reply:
Ah, I must've uploaded the wrong file or something. Now the correct file should be there.